optaweb/image.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "rhdm-7/rhdm70-optaweb"
+description: "Red Hat Decision Manager Employee Rostering Optimizer 7.0 container image"
```

Comment:
The naming is not canonical.
Parts of the name:
- Company: Red Hat
- SKU: Decision Manager or PAM (and both are a bit misleading)
- Product: Business Optimizer
- App: OptaWeb Employee Rostering
- Suffix: container image

Suggested names.
Proposal A) "Red Hat Decision Manager OptaWeb Employee Rostering container image"
Proposal B) "Red Hat Business Optimizer OptaWeb Employee Rostering container image"
I prefer B, if we can get away with not mentioning the SKU name :)

Reply:
We are targeting 7.1 release, there should be "Optimizer 7.1 container image".

Reply:
changed to Red Hat Business Optimizer OptaWeb Employee Rostering 7.1 container image
updated all 7.0.1 to 7.1.0 as suggested